### ISSUE TITLE
Implement JAX-based force computation experiment

### ISSUE THREAD

**Issue by stevemikedan** [stevemikedan/CosmoSim#30]
Rebuilt gravitational force calculation using jax.numpy and tested accuracy.
